src/sentry/incidents/logic.py

```diff
@@ -662,6 +663,20 @@ def create_alert_rule(
     return alert_rule
 
 
+def subscribe_projects_to_alert_rule(
```

Comment:
This is just undoing the code in the same file in https://github.com/getsentry/sentry/pull/65837/files